test/runtests.jl

```diff
@@ -271,7 +275,7 @@ end
 
 @test_zip [1,2,3] [:a, :b, :c] ['x', 'y', 'z']
 @test_zip [1,2,3] [:a, :b] ['w', 'x', 'y', 'z']
-@test_zip [1,2,3] @compat(Union{})[] ['w', 'x', 'y', 'z']
```

Comment:
The type of the empty array is not material to the test. It was only changed from `[]` to avoid a deprecation warning from https://github.com/JuliaLang/julia/pull/8578.